docs/blog/posts/introducing-structured-outputs.md

```diff
@@ -0,0 +1,468 @@
+---
+draft: False
+date: 2024-08-20
+slug: is-instructor-dead
+tags:
+  - OpenAI
+authors:
+  - ivanleomk
+---
+
+# Is Instructor Dead?
+
+## What's Open AI's Structured Output mode all about?
+
+OpenAI's new Structured Output mode is a huge step change for developers building complex workflows. Given an arbitrary JSON Schema, Structured Output ensures that the response matches the schema exactly.
+
+Here's a basic example.
+
+```python
+import openai
+from pydantic import BaseModel
+
+
+class User(BaseModel):
+    name: str
+    age: int
+
+
+client = openai.OpenAI()
+resp = client.beta.chat.completions.parse(
+    response_format=User,
+    messages=[
+        {
+            "role": "user",
+            "content": "Extract the following user: Jason is 25 years old.",
+        },
+    ],
+    model="gpt-4o-mini",
+)
+
+print(resp.choices[0].message.parsed)
+#> name='Jason' age=25
+```
+
+With guaranteed schema adherence, outputs always conform to your defined Pydantic model, eliminating type mismatches and missing fields. However, while Structured Outputs solve many common issues, two key challenges emerge when building more sophisticated applications - that of Validation and Streaming.
```

Comment:
move this to top but theres more issues than these two, like latency, and also use a list